### PR TITLE
toasted activities extend to mobile view

### PR DESCRIPTION
**Pull request by vanillaHill**
https://github.com/FortressFinance/issues/issues/220

### CODE REVIEW

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -43,27 +44,14 @@ const Layout: FC<PropsWithChildren> = ({ children }) => {
   ])
 
   const router = useRouter()
+  const mdScreen = useMedia("(max-width: 768px)")
```
Comment (aquaticone) [inline src/components/Layout.tsx:47]: This is really bad for performance. I think you should just do this if possible

```tsx
<div className="max-md:hidden">
  {/* desktop toaster */}
</div>
<div className="md:hidden">
  {/* mobile toaster */}
</div>
```

If that doesn't work we can leave this for now but we'll want to write a custom toaster. I always forget that react-hot-toast doesn't have proper responsive styling.

**Review by aquaticone — APPROVED** [review]
Approving so you can merge this when you're ready but I'd like you to try one thing first

File: src/components/Layout.tsx (hunk 1 above)
Comment (vanillaHill) [reply src/components/Layout.tsx:47]: Already tried this scenario, it didn't work. I'll redo it again later, maybe I missed something.

Comment (aquaticone) [reply src/components/Layout.tsx:47]: No worries sir feel free to merge.

Comment (vanillaHill) [reply src/components/Layout.tsx]: It's working, pushed the changes.

**Pull request merged**